Documentation/openshiftdocs/modules/nodeexportercollectorconfig.adoc

```diff
@@ -0,0 +1,25 @@
+// DO NOT EDIT THE CONTENT IN THIS FILE. It is automatically generated from the 
+	// source code for the Cluster Monitoring Operator. Any changes made to this 
+	// file will be overwritten when the content is re-generated. If you wish to 
+	// make edits, read the docgen utility instructions in the source code for the 
+	// CMO.
+	:_content-type: ASSEMBLY
+
+== NodeExporterCollectorConfig
+
+=== Description
+
+The `NodeExporterCollectorConfig` resource defines settings for individual collectors of the Node Exporter component.
+
+
```

Comment:
```suggestion
```

Reply:
The `*.adoc` files are generated from `pkg/manifests/types.go`. It seems the number of blank lines here is not configurable. 😅